src/batterymonitor.h

```diff
@@ -83,6 +83,10 @@ class BatteryMonitor
     float voltage = -1;
     float level = -1;
 
+public:
+    float getVoltage() const { return voltage; }
+    float getLevel() const { return level; }
+
```

Comment:
Please move the public headers together.

Reply:
done.